packages/fast-components-react-base/src/radio/radio.props.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { IManagedClasses, IRadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface IRadioHandledProps {
+
+    /**
+     * Unique Id
+     */
+    id: string;
+
+    /**
+     * The checked state
+     */
+    checked?: boolean;
+
+    /**
+     * The disabled state
+     */
+    disabled?: boolean;
+
+    /**
+     * The onChange state
+     */
+    onChange?: boolean;
+
+    /**
+     * The radio content
+     */
+    children?: React.ReactNode;
+}
+
+export interface IRadioUnhandledProps extends React.AllHTMLAttributes<HTMLElement> {}
```

Comment:
@nicholasrice 's PR just went in that changes how this is done a bit. I think you need to update before it can go in

Reply:
@nalogiudice That work went into the v3 components branch. I think most of the work to align will need to be done in that branch or after it's merged in.

Reply:
yep - I will make the relevant updates when I rebase the 3.0.0 branch with this change so the commit history is accurate.

Reply:
@chrisdholt and @nicholasrice sorry guys - thought that went into master -disregard